addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientDetailsEdit.js

```diff
@@ -49,8 +49,18 @@ odoo.define('point_of_sale.ClientDetailsEdit', function(require) {
                     processedChanges[key] = value;
                 }
             }
-            if ((!this.props.partner.name && !processedChanges.name) ||
-                processedChanges.name === '' ){
+
+            let savedName = this.props.partner.name;
+            let newName = processedChanges.name;
+            let isMissingName = false;
+            // if invalid name in db  (undefined or empty/blank string)
+            if (!savedName || (savedName && savedName.trim() === '') ) 
+                isMissingName = true;
+            // if new input, check that that it is valid
+            if (typeof newName === 'string') 
+                isMissingName = newName.trim() === ''? true: false;
```

Comment:
you can just do: `isMissingName = newName.trim() === ''`

Reply:
thanks!!